### DISCUSSION TITLE
Invite returns 404 if repository changed its name

### DISCUSSION THREAD

**Discussion by acerbisgianluca** [community/community#63493] [Repositories]
### Select Topic Area

Bug

### Body

Hey there, I just got an invite to join a repository that changed its name after the invite has been sent, but when I click on the notification, the page points to the old repository name (`https://github.com/ORG/OLD_REPO_NAME/invitations`) and it returns a 404 error. Manually fixing the URL works.
In the image, you can see that the notification comes from the new repository, but the content of the notification itself still points to the old name (I changed it for privacy).
![image](https://github.com/community/community/assets/22961967/aaf37b85-5345-4964-bc73-ba8bcd2b7570)

**Comment by EdamAme-x** [comment]
It seems difficult in terms of compatibility...

However, it is an important matter related to the core. I hope it will be solved...
